src/ui/Windows/Fonts.cpp

```diff
@@ -191,10 +191,25 @@ void Fonts::SetTreeListFont(LOGFONT *pLF, const int iPtSz)
   }
 
   if (iPtSz == 0) {
```

Comment:
Maybe extract this if-else into a helper method that returns a scaled `LOGFONT`, so `SetTreeListFont` could remain focused on one thing - setting `m_pTreeListFont`?